xmake/rules/windows/xmake.lua

```diff
@@ -0,0 +1,42 @@
+rule("win.subsystem.console")
+    add_deps("win.subsystem")
+    on_load(function(target)
+        target:data_set("win.subsystem", "console")
```

Comment:
remove these rules. use `set_values("windows.subsystem", "console")` + platform.windows.subsystem rule instead of it.